### PR TITLE
cloud changefeed: add cloud changefeed states

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added a new section about TiDB Cloud changefeed states.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-changefeed-states.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+summary: Learn the meaning of each changefeed state in TiDB Cloud.
+---
+
+# Changefeed States
+
+A [changefeed](/tidb-cloud/changefeed-overview.md) is a replication task in TiCDC, which replicates the data change logs of specified tables in a TiDB cluster to the designated downstream. You can run and manage multiple changefeeds in a TiCDC cluster.
+
+The state of a replication task represents the running status of the replication task. During the running of TiCDC, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to the change of the replication task state. This document describes the states of TiCDC replication tasks and the transfer relationships between states.
+
+The states in the preceding state transfer diagram are described as follows:
```
Comment (Frank945946) [inline tidb-cloud/tidb-cloud-changefeed-states.md:12]: There is no diagram.Only status description.

File: tidb-cloud/tidb-cloud-changefeed-states.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- `RUNNING`：the replication task runs normally and the checkpoint-ts proceeds normally.
+- `FAILED`: the replication task fails. Due to some unrecoverable errors, the replication task cannot resume and cannot be recovered. The changefeed in this state does not block GC operations.
+- `CREATING`: the replication task is being created.
+- `RESUMING`: the replication task is being resumed.
+- `PAUSING`: the replication task is being paused.
+- `PAUSED`: the replication task is paused.
+- `DELETING`: the replication task is being deleted.
+- `DELETED`: the replication task is deleted.
+- `EDITING`: the replication task is being edited.
+- `WARNING`: the replication task returns an error. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `Normal`. The changefeed in this state blocks GC operations.
```
Comment (Frank945946) [inline tidb-cloud/tidb-cloud-changefeed-states.md:23]: Modify the order, as following:
Creating\running\editing\\pausing\paused\resuming\deleting\deleted\warning\failed

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-changefeed-states.md (hunk 1 above)
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-changefeed-states.md:12]:
```suggestion
The states are described as follows:
```

File: tidb-cloud/tidb-cloud-changefeed-states.md (hunk 2 above)
Comment (hfxsd) [reply tidb-cloud/tidb-cloud-changefeed-states.md:22]: updated

File: tidb-cloud/changefeed-overview.md [hunk 3]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
 ## Changefeed billing
 
 To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md).
+
+## Changefeed states
+
+The state of a replication task represents the running state of the replication task. During the running of TiCDC, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to the change of the replication task state. 
+
+This section describes the states of changefeed replication tasks. The states are described as follows:
```
Comment (hfxsd) [inline tidb-cloud/changefeed-overview.md:71]:
```suggestion
The states are described as follows:
```

File: tidb-cloud/changefeed-overview.md [hunk 4]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
 ## Changefeed billing
 
 To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md).
+
+## Changefeed states
+
+The state of a replication task represents the running state of the replication task. During the running of TiCDC, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to the change of the replication task state.
```
Comment (qiancai) [inline tidb-cloud/changefeed-overview.md:69]:
```suggestion
The state of a replication task represents the running state of the replication task. During the running process, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to changes of the replication task state. 
```

File: tidb-cloud/changefeed-overview.md [hunk 5]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
...
 To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md).
+
+## Changefeed states
+
+The state of a replication task represents the running state of the replication task. During the running of TiCDC, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to the change of the replication task state. 
+
+The states are described as follows:
+
+- `CREATING`: the replication task is being created.
+- `RUNNING`：the replication task runs normally and the checkpoint-ts proceeds normally.
```
Comment (qiancai) [inline tidb-cloud/changefeed-overview.md:74]:
```suggestion
- `RUNNING`: the replication task runs normally and the checkpoint-ts proceeds normally.
```

File: tidb-cloud/changefeed-overview.md [hunk 6]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
...
+- `EDITING`: the replication task is being edited.
+- `PAUSING`: the replication task is being paused.
+- `PAUSED`: the replication task is paused.
+- `RESUMING`: the replication task is being resumed.
+- `DELETING`: the replication task is being deleted.
+- `DELETED`: the replication task is deleted.
+- `WARNING`: the replication task returns an error. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `Normal`. The changefeed in this state blocks GC operations.
```
Comment (qiancai) [inline tidb-cloud/changefeed-overview.md:81]:
```suggestion
- `WARNING`: the replication task returns an error. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `RUNNING`. The changefeed in this state blocks GC operations.
```

Comment (qiancai) [reply tidb-cloud/changefeed-overview.md:81]: returns an error 还是 returns an warning 呢

Comment (hfxsd) [reply tidb-cloud/changefeed-overview.md:81]: warning. fixed

File: tidb-cloud/changefeed-overview.md [hunk 7]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
...
+
+- `CREATING`: the replication task is being created.
+- `RUNNING`: the replication task runs normally and the checkpoint-ts proceeds normally.
+- `EDITING`: the replication task is being edited.
+- `PAUSING`: the replication task is being paused.
+- `PAUSED`: the replication task is paused.
+- `RESUMING`: the replication task is being resumed.
+- `DELETING`: the replication task is being deleted.
+- `DELETED`: the replication task is deleted.
+- `WARNING`: the replication task returns an warning. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `RUNNING`. The changefeed in this state blocks GC operations.
```
Comment (qiancai) [inline tidb-cloud/changefeed-overview.md:81]:
```suggestion
- `WARNING`: the replication task returns a warning. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `RUNNING`. The changefeed in this state blocks GC operations.
```

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-changefeed-states.md (hunk 1 above)
Comment (hfxsd) [reply tidb-cloud/tidb-cloud-changefeed-states.md]: fixed

File: tidb-cloud/changefeed-overview.md [hunk 8]
```diff
@@ -63,3 +63,20 @@ To create a changefeed, refer to the tutorials:
 ## Changefeed billing
 
 To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md).
+
+## Changefeed states
+
+The state of a replication task represents the running state of the replication task. During the running process, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to changes of the replication task state. 
+
+The states are described as follows:
```
Comment (hfxsd) [inline tidb-cloud/changefeed-overview.md:71]:
```suggestion
The states are described as follows: 
```

**Pull request merged**